### PR TITLE
Pickup and apply "section.isExpanded"

### PR DESCRIPTION
**Pull request by samuelmale**
## Requirements

- [X] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [X] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary

Pickup and apply "isExpanded" property from the form section as defined in the schema

### CODE REVIEW

File: src/components/page/ohri-form-page.tsx [hunk 1]
```diff
@@ -1,33 +1,28 @@
...
 import styles from './ohri-form-page.scss';
 
-function OHRIFormPage({ page, onFieldChange, setSelectedPage, isCollapsed }) {
-  let newLabel = page.label.replace(/\s/g, '');
-
-  const handleEnter = elementID => {
-    setSelectedPage(elementID);
-  };
-
+function OHRIFormPage({ page, onFieldChange, setSelectedPage, isFormExpanded }) {
```
Comment (samuelmale) [inline src/components/page/ohri-form-page.tsx:8]: Working on a more ideal typing in separate PR

**Review by donaldkibet — APPROVED** [review]
LGTM Thanks @samuelmale

**Review by pirupius — APPROVED** [review]
LGTM

**Review by pirupius — APPROVED** [review]
LGTM

**Pull request merged**